yang/frr-zebra.yang

```diff
@@ -2342,6 +2342,17 @@ module frr-zebra {
             "RFC 4861: Neighbor Discovery for IP version 6 (IPv6)
                        - AdvOtherConfigFlag";
         }
+        leaf link-mtu {
+          type uint32;
+          default "0";
+          description
+            "The value to be placed in MTU options sent by the
+            router.  A value of zero indicates that no MTU options
+            are sent.";
```

Comment:
Is this copied from an IETF model? Why don't we use the lack of the leaf's presence in the data to mean not option sent? Seems more elegant than an active value of 0.

Reply:
Yes, it's from an IETF model. This leaf directly controls the `AdvLinkMTU` variable which is treated in the code depending on zero/non-zero. Removing the default value would just mean an additional `destroy` callback which would set the `AdvLinkMTU` to zero. I don't think it makes sense, especially with the deviation from the standard model.